Update the-box-model lesson. Add hint on how to use Dev Tool to perform the suggested change from the curriculum. (#27955)
TheOdinProject/curriculum: foundations/html_css/css-foundations/the-box-model.md

![boxes](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/css-foundations/the-box-model/imgs/boxes.png)

> You can use the browser's inspector to add the CSS above to this web page if you want. Boxes in boxes!
> Hint: use the + button at top of Styles-panel on Elements tab to add new CSS rule.

Reviewer: ```suggestion
You can use the browser's inspector to add the CSS above to this web page if you want by clicking the `+` button in the top right of the "Styles" panel within the "Elements" tab. Boxes in boxes!
```
Just a slight wording change suggestion but otherwise, I think it's fair to be a little more explicit with how to do this step!

Author: Hi thank you for your review. My reasoning to separate the sentence was to entice the reader to be more proactive in their learning and thus to reward those with more curiosity to discover the solution as "their own solution". Rather than just following a "complete" instruction.
But either way, I don't have much experience yet in this learning and teaching field, so whatever you think is best I don't mind :)

Reviewer: I think for this specific step, combining them will be fine and perhaps let the sentence flow a little smoother.

Author: Ok that is fine. Anything I need to do from my end to proceed?

Reviewer: You just need to make the change, commit then push it. In this instance because my comment includes an embedded suggestion, you should be able to hit "commit suggestion" to do so directly from that if you're happy with it. Otherwise if you still wish to tweak the wording yourself, you'll just have to commit and push your change manually.
